lib/cubits/github/github_states.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:equatable/equatable.dart';
+import 'package:hacki/models/github/github_repository.dart';
+
+enum GithubStatus { initial, loading, success, failure }
+
+class GithubState extends Equatable {
+  const GithubState({
+    this.repositories = const <String, GithubRepository>{},
+    this.status = GithubStatus.initial,
+    this.error,
+  });
+
+  final Map<String, GithubRepository> repositories;
```

Comment:
I think there should be just one repository for each state since each post only includes one url.

Reply:
I checked that coupling this with existing repo seemed to break the functioning and increase time lag